openedx/frontend-app-course-authoring
# [UU-58] Implement tagging & taxonomy feature in outline

### Description

Adds the `Manage tags` menu item and `TagCount` button to Units. 

### Screenshots

![image](https://github.com/openedx/frontend-app-course-authoring/assets/11827305/4143b0c8-9833-4351-9075-bf0e31023c88)

### Supporting information

- [Requirements](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3866460192/New+Additions+to+Course+Content+Outline#Tagging-%26-Taxonomy)
- Internal ticket: [BB-8633](https://tasks.opencraft.com/browse/BB-8633)

### Testing instructions

- Enable `contentstore.new_studio_mfe.use_new_course_outline_page` waffle flag
- Run [Taxonomy Sample Data](https://github.com/open-craft/taxonomy-sample-data) script to create units with tags.
- Go to a course to display the course outline.
- Expand sections and subsections to show units.
- Verify the tag count component.
- Click on the tago count component. Verify that it opens the tag drawer.
- Add or delete tags and verify that the tag count is updated in real time.
- Close the drawer.
- Click on the menu dots an on `Manage tags`. Verify that it opens the tag drawer.
